css/class-naming-conventions/readme.md

```diff
@@ -43,23 +44,23 @@ Our convention (which we call CSM or Component, Sub-Component, Modifier) uses [B
 
 This example may seem confusing at first but if we break down each of the selectors that we have, it begins to make more sense.
 
-`.c-blog` This is a component. It describes a high level module or component. In this instance, it describes the container for all of our blog posts.
+`c-blog` This is a component. It describes a high level module or component. In this instance, it describes the container for all of our blog posts.
 
-`.c-blog__title` This is a sub-component. It's always a child of a module or component. In this instance, it is a title for our blog post container
+`c-blog__title` This is a sub-component. It's always a child of a module or component. In this instance, it is a title for our blog post container
 
-`.c-blog-post` This is another component. This one describes a specific blog post. We make this its own component because a blog post is not necessarily a child of the blog container. It can and should be able to live independently.
+`c-blog-post` This is another component. This one describes a specific blog post. We make this its own component because a blog post is not necessarily a child of the blog container. It can and should be able to live independently.
```

Comment:
Is it true that `c-blog-post` might not always be a child of `c-blog`? I was under the impression that sub-components always had to live inside the component itself–unless what we're trying to say here is that it doesn't have to be a direct child of `c-blog`.